mockserver-spring/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.mock-server</groupId>
+        <artifactId>mockserver</artifactId>
+        <version>5.11.1-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>mockserver-spring</artifactId>
+    <name>MockServer Spring</name>
+    <description>A module containing helpers for writing mockserver tests in spring projects</description>
+    <url>http://www.mock-server.com</url>
+
+    <properties>
+        <spring-boot.version>2.2.8.RELEASE</spring-boot.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
```

Comment:
Versions should be manged in the parent pom

Reply:
done